packages/abstractions/.github/workflows/create-release.yml

```diff
@@ -0,0 +1,40 @@
+name: "Tag & create GitHub release"
+
+on:
+  push:
```

Comment:
you're going to be facing an issue here. 
Because the merge was done by GitHub actions, it won't trigger further actions to avoid overwhelming the service.
https://github.com/orgs/community/discussions/25702

Reply:
Thanks for pointing this out. I've changed this so that the GitHub app enables auto-merge during the split after creating a PR in the target repo. This can trigger a subsequent workflow - tested [here](https://github.com/sample-monorepo/sample-http/pull/18) & the release triggered [here](https://github.com/sample-monorepo/sample-http/actions/runs/13343681409)